tests/test_kernels.hh

```diff
@@ -107,9 +108,12 @@ namespace rascal {
                  {"initialization_arguments",
                   {{"cutoff", cutoff},
                    {"consider_ghost_neighbours", consider_ghost_neighbours}}}};
+        json ad1b{{"name", "AdaptorCenterContribution"},
+                  {"initialization_arguments", {}}};
         json ad2{{"name", "AdaptorStrict"},
                  {"initialization_arguments", {{"cutoff", cutoff}}}};
         adaptors.emplace_back(ad1);
+        adaptors.emplace_back(ad1b);
```

Comment:
I did not get the name "b for bis". In the tests can you even write out what ad1b or make comment what this means.

Reply:
these are dummy variable that are just here for convenience since I could not find how to make a json::array of dictionaries through the constructor. Is it really essential to have meaningful variable name even when the variable are not ?

Reply:
no, the name just jumped out of the pattern and I thought there would be some essential information in the name which I do not get